packages/workbox-core/test/bundle/node/utils/test-logHelper.js

```diff
@@ -0,0 +1,339 @@
+import chai from 'chai';
+import sinon from 'sinon';
+
+import logHelper from '../../../../src/utils/logHelper';
+
+const expect = chai.expect;
+
+describe(`logHelper [${process.env.NODE_ENV}]`, function() {
+  let stubs = [];
+
+  beforeEach(function() {
+    logHelper.logLevel = logHelper.LOG_LEVELS.verbose;
+  });
+
+  afterEach(function() {
```

Comment:
We could get into the habit of using Sinon's Sandboxes instead of our more manual approach with the global stubs array:

https://stackoverflow.com/a/14041593/385997